bugsnag-android-core/src/main/java/com/bugsnag/android/internal/JsonHelper.kt

```diff
@@ -32,14 +32,14 @@ class JsonHelper private constructor() {
                 bytes,
                 bytes.size
             )
-            require(!document.isNullOrEmpty()) { "JSON document is invalid" }
+            require(document != null) { "JSON document is invalid" }
```

Comment:
nit: using `requireNotNull` in this class would be the idiomatic Kotlin way of doing things